client/src/app/routes/IdentityResolver.tsx

```diff
@@ -190,81 +190,84 @@ class IdentityResolver extends AsyncComponent<
-                                                        </div>
-                                                    )}
-
-                                                    {!this.state.error &&
-                                                        !(this.state.latestMessageId === this.EMPTY_MESSAGE_ID) && (
-                                                            <MessageTangleState
-                                                                network={this.props.match.params.network}
-                                                                status={this.state.messageTangleStatus}
-                                                                milestoneIndex={
-                                                                    this.state.metadata?.referencedByMilestoneIndex ??
-                                                                    this.state.metadata?.milestoneIndex
-                                                                }
-                                                                onClick={this.state.metadata?.referencedByMilestoneIndex
-                                                                    ? (messageId: string) => this.props.history.push(
-                                                                        `/${this.props.match.params.network
-                                                                        }/search/${messageId}`)
-                                                                    : undefined}
-                                                            />
-                                                        )}
+                                        <div>
+                                            {this.state.version && this.state.version === "legacy" && (
+                                                <div className="section">
+                                                    <div className="legacy-method">
+                                                        <span>
+                                                            This DID was created by a deprecated version of the identity
+                                                            {" "}
+                                                            library. If this is your DID you can recreate it using the
+                                                            {" "}
+                                                        </span>
+                                                        <a
+                                                            href="https://github.com/iotaledger/identity.rs/releases"
+                                                            target="_blank"
+                                                            rel="noreferrer"
+                                                        >
+                                                            latest version
+                                                        </a>.
```

Comment:
Maybe we can change the color of the a-tag to inform the user that it's a link.

![image](https://user-images.githubusercontent.com/52735957/153057649-e1631110-a951-42a2-bc59-5f7b0750645d.png)

Reply:
pushed